tests/test_cli.py

```diff
@@ -50,10 +52,23 @@ class mod:
 
 
 def test_prepare_exec_for_file(test_apps):
-    assert prepare_exec_for_file('test.py') == 'test'
+    """Expects correct path to be set and correct module name to be returned.
+
+    :func:`prepare_exec_for_file` has a side effect, where
+    parent directory of given file is added to `sys.path`.
+    """
+    realpath = os.path.realpath('/usr/share/test.py')
```

Comment:
I'd use `/tmp` as an example path. IMHO that makes more sense than a path that'd probably never contain a python file